### PR TITLE
🎨 👽️ allow user to unhide study/solver job

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
- Allow users to unhide (in the ui) solver/study jobs.

## Related issue/s

- fixes https://github.com/ITISFoundation/osparc-issues/issues/1469

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by wvangeit — APPROVED** [review]
Thx. Very useful!

File: services/api-server/src/simcore_service_api_server/services/webserver.py [hunk 1]
```diff
@@ -244,11 +244,13 @@ async def update_me(self, profile_update: ProfileUpdate) -> Profile:
     # PROJECTS -------------------------------------------------
 
     @_exception_mapper({})
-    async def create_project(self, project: ProjectCreateNew) -> ProjectGet:
+    async def create_project(
+        self, project: ProjectCreateNew, hidden: bool
```
Comment (pcrespov) [inline services/api-server/src/simcore_service_api_server/services/webserver.py:248]: MINOR: rename `is_hidden` and enforce key argument
```python
async def foo(self, project: ProjectCreateNew, *, is_hidden: bool)
    ...
```

File: services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py [hunk 2]
```diff
@@ -81,8 +81,9 @@ async def create_study_job(
     job_inputs: JobInputs,
     webserver_api: Annotated[AuthSession, Depends(get_webserver_session)],
     url_for: Annotated[Callable, Depends(get_reverse_url_mapper)],
+    hidden: Annotated[bool, Query()] = True,
```
Comment (pcrespov) [inline services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py:84]: Add a bit of human-readable doc here :-)  It is not obvious what is a "hidden job"

**Review by pcrespov — APPROVED** [review]
thx. 
Left a few suggestions

**Review by sanderegg — APPROVED** [review]

File: services/api-server/src/simcore_service_api_server/services/webserver.py (hunk 1 above)
Comment (bisgaard-itis) [reply services/api-server/src/simcore_service_api_server/services/webserver.py:248]: yep, good point. Done

File: services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py (hunk 2 above)
Comment (bisgaard-itis) [reply services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py:84]: 👍

**Pull request merged**